ingestion/src/levels.rs

```diff
@@ -77,14 +113,16 @@ pub async fn fetch_levels(stinfo_conn_string: &str) -> Result<ParamLevelTable, E
 
     for row in rows {
         let hlevel_scale: i32 = row.get(1);
-        // currently only have 0 and -2, aka m and cm
-        param_level.insert(
-            row.get(2),
-            Level {
-                hlevel: row.get(0),
-                hlevel_scale: Some(hlevel_scale),
-            },
-        );
+        if ![0, -2].contains(&hlevel_scale) {
+            // currently only have 0 and -2, aka m and cm
+            param_level.insert(
+                row.get(2),
+                Level {
+                    hlevel: row.get(0),
+                    hlevel_scale: Some(hlevel_scale),
+                },
+            );
+        }
```

Comment:
Since we mentioned wanting to log this:

```suggestion
        } else {
            error!("Invalid hlevel_scale found in stinfosys: {}", hlevel_scale);
        }
```

Probably also need
```
import tracing::error;
```
at the top

Reply:
Thanks! Done